modules/terraform-cdp-aws-pre-reqs/defaults.tf

```diff
@@ -50,7 +35,7 @@ locals {
   security_group_rules_ingress = [
```

Comment:
@jimright Taking a second look, I expected to see two SGs here - see https://cloudera.github.io/cdp-dev-docs/cli-docs/environments/create-aws-environment.html#:~:text=Creates%20a%20new%20AWS%20environment,when%20an%20environment%20is%20created.

> securityGroupIdForKnox -> (string)
> Security group where Knox-enabled hosts are placed. Mutually exclusive with cidr.
> 
> defaultSecurityGroupId -> (string)
> Security group where all other hosts are placed. Mutually exclusive with cidr.

How do we handle this in create\_vpc = true. Do we allow bringing existing security group?

Reply:
> I expected to see two SGs here

Yes, we do create 2 SGs for CDP - one default and one for Knox. See code for this in https://github.com/cloudera-labs/terraform-cdp-modules/blob/42308bd0dd3d51e1100caf6a2457bb261b45703b/modules/terraform-cdp-aws-pre-reqs/main.tf#L45-L132.

> How do we handle this in create_vpc = true. Do we allow bringing existing security group?

SGs are still created regardless of the value of create_vpc.
We don't currently support byo-SG but as part of the creation of more fine-grained modules this should be possible.